ftw/book/tests/test_uninstall.py

```diff
@@ -17,7 +17,6 @@ class TestGenericSetupUninstall(TestCase, GenericSetupUninstallMixin):
     @property
     def skip_files(self):
         if getFSVersionTuple() < (4, 3):
```

Comment:
The whole method can be simplified because there's no longer the need for testing if we're running Plone < 4.3.